pkg/scheduler/scheduler.go

```diff
@@ -480,23 +503,26 @@ func (s *Scheduler) applyAdmissionWithSSA(ctx context.Context, w *kueue.Workload
 	return workload.ApplyAdmissionStatus(ctx, s.client, w, false)
 }
 
-type entryOrdering []entry
+type entryOrdering struct {
+	entries          []entry
+	workloadOrdering workload.Ordering
+}
 
-func (e entryOrdering) Len() int {
```

Comment:
nit: I would suggest reverting the renames, but feel free to keep

Reply:
I also think that the previous receiver name, `e` would be better.

Reply:
Done